mh/pat_item.py

```diff
@@ -260,13 +260,15 @@ def pack_binary(s):
     return struct.pack(">BH", ItemType.Binary, len(s)) + s
 
 
-def unpack_binary(data, offset=0):
+def unpack_binary(data, offset=0, pattern=None):
```

Comment:
Yeah, this `unpack_***`` are only meant to be used with the `PatData` structures.

Reply:
Exactly, `struct.unpack_from` can be used directly instead. This `pat_item` module is for PAT items, the code should inserted into a relevant module if it exists, otherwise one should be created.